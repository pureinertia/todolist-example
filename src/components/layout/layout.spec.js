import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Layout from './layout';

Enzyme.configure({ adapter: new Adapter() });

describe('<Layout />', () => {
  const props = {
    user: {
      name: 'Some Name',
    },
  };

  it('renders without errors', () => {
    const wrapper = shallow(<Layout {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
