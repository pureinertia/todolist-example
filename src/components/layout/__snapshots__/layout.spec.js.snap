// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> renders without errors 1`] = `
<div
  className="layout"
>
  <div
    className="layout--side"
  >
    <ConnectFunction />
  </div>
  <section
    className="layout--main"
  />
</div>
`;
